tests/acceptance/fixtures/responses/universal.js

```diff
@@ -0,0 +1,486 @@
+import { RequestMock } from 'testcafe';
+
+const UniversalSearchResponse = {
+  meta: {
+    uuid: '0180235e-7fcd-5262-247b-d3537eea0002',
+    errors: []
+  },
+  response: {
+    businessId: 3350634,
+    modules: [
+      {
+        verticalConfigId: 'people',
+        resultsCount: 1,
+        encodedState: '',
+        results: [
+          {
+            data: {
+              id: 'Employee-2116',
+              type: 'ce_person',
+              website: 'http://www.test.com',
+              address: {
+                line1: '7900 Westpark Drive',
+                city: 'Mclean',
+                region: 'VA',
+                postalCode: '22102',
+                countryCode: 'US'
+              },
+              associations: [
+                'Runners Association'
+              ],
+              description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Mollis nunc sed id semper risus. Commodo elit at imperdiet dui accumsan sit amet nulla facilisi. Scelerisque viverra mauris in aliquam. A erat nam at lectus. Egestas purus viverra accumsan in nisl nisi scelerisque. Habitant morbi tristique senectus et netus. Congue nisi vitae suscipit tellus mauris a diam maecenas sed.',
+              name: 'Amani Farooque',
+              cityCoordinate: {
+                latitude: 38.936519622802734,
```

Comment:
I'm wondering if we want to scramble this personal info at all? Some of it looks like real data